src/denovo_discovery/discover_main.cpp

```diff
@@ -88,12 +88,62 @@ void setup_discover_subcommand(CLI::App& app)
             "Use binomial model for kmer coverages [default: negative binomial]")
         ->group("Parameter Estimation");
 
+    discover_subcmd
+        ->add_flag("--dont-auto-update-params", opt->do_not_auto_update_params,
+            "By default, pandora automatically updates error rate and kmer coverage model parameters based "
+            "on the mapping of the previous sample. This could potentially generate "
+            "more accurate results if your samples have no sequencing issues and a "
+            "consistent protocol was followed for the sequencing of all samples. If this is "
+            "not the case, deactivate this feature by activating this flag")
+        ->group("Parameter Estimation");
+
     discover_subcmd
         ->add_option("--max-covg", opt->max_covg, "Maximum coverage of reads to accept")
         ->capture_default_str()
         ->type_name("INT")
         ->group("Filtering");
 
+    discover_subcmd
+        ->add_option(
+            "--min-abs-gene-coverage", opt->min_absolute_gene_coverage,
+            "Minimum absolute mean gene coverage to keep a gene. Given the "
+            "coverage on the kmers of the maximum likelihood path of a gene, we compute "
+            "the mean gene coverage and compare with the value in this "
+            "parameter. If the mean is lower than this parameter, "
+            "the gene is filtered out, e.g. if this parameter value is "
+            "3, then all genes with mean <3 will be filtered out.")
```

Comment:
if this value is here; my guess is that many reads get annotated with a gene in the SAM file during mapping, but then pandora looks across all the information, and then decides whether a gene is truly present. If so, and pandora has decided the gene is not there, @Danderson123 might need to remove those gene annotations from his reads.

Reply:
Yeah, AFAIK @Danderson123 just processes the SAM file, and the SAM file is just read mapping, there is no FP filtration, FP gene filtration happens later

Reply:
ok so this is not relevant to this PR, but is a reminder to @Danderson123 that if, when he takes the pandora SAM, he also parses the gene presence file, he will get the clean filtered list of genes present, and can make a dictionary of them , and when parsing the SAM, can ignore genes on reads which are not in the dictionary